lms/templates/main.html

```diff
@@ -62,7 +63,13 @@
   <link rel="icon" type="image/x-icon" href="${static.url(microsite.get_value('favicon_path', settings.FAVICON_PATH))}" />
 
   <%static:css group='style-vendor'/>
-  <%static:css group='style-main'/>
+  ## route around the overwhelmingly complicated static:css nonsense
```

Comment:
Does this substantial-looking change require any review by RTL-focused folks (@sarina) or any updates to documentation?